src/unix/sunos.c

```diff
@@ -546,8 +546,11 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   }
 
   if (first_run) {
-    uv__io_init(&handle->loop->fs_event_watcher, uv__fs_event_read, portfd);
-    uv__io_start(handle->loop, &handle->loop->fs_event_watcher, POLLIN);
+    return uv__io_init_start(handle->loop,
+                             &handle->loop->fs_event_watcher,
+                             uv__fs_event_read,
+                             portfd,
+                             POLLIN);
```

Comment:
Should undo the effects of `uv__handle_start()` and I don't know what `uv__fs_event_rearm()` does but that may have to be undone as well.

Reply:
Calling `uv__handle_stop` is our undo, right?